src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RATE = 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_RENTAL = 7;
+  const SHORT_TERM_RENTAL = 3;
+
+  if (days >= LONG_TERM_RENTAL) {
+    return (DAILY_RATE * days) - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= SHORT_TERM_RENTAL) {
+    return (DAILY_RATE * days) - SHORT_TERM_DISCOUNT;
+  }
+
+  return DAILY_RATE * days;
```

Comment:
The final return statement correctly handles the case where no discount applies.